tests/test_question.py

```diff
@@ -62,3 +62,21 @@ def test_async_ask_question():
         assert response == "World"
     finally:
         inp.close()
+
+
+def test_multiline_text():
```

Comment:
can we also add a test to check the other key combination to submit the input, `Alt+Enter`?

Reply:
[It seems it's the same](https://python-prompt-toolkit.readthedocs.io/en/master/pages/advanced_topics/key_bindings.html#binding-alt-something-option-something-or-meta-something).